components/CardModule.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div class="flex flex-col h-full">
+    <div class="relative flex flex-1 flex-col space-y-2 px-6 py-8 group">
+      <a :href="module.website" :aria-label="module.website" target="_blank" rel="noopener" class="absolute inset-0" />
+      <div class="transition-opacity duration-200 ease-in-out opacity-0 group-hover:opacity-100 absolute top-4 right-6 cursor-pointer">
+        <img alt="website" src="~/assets/icons/ext.svg" width="24" height="24">
+      </div>
+
+      <nuxt-image
```

Comment:
I think svg is now supported by image module. /cc @farnabaz

Reply:
We can integrate it with this [PR](https://github.com/nuxt/modules/pull/45) no ?

Reply:
Yeah we may simply upgrade in this PR :) Since otherwise one will conflict

Reply:
I integrated it so you can closed #45 
Maybe in another PR, we could manage all images with `nuxt-image` directly ?

Reply:
Thanks man! For rest of images i would suggest waiting for feature `nuxt-img` landing it is specially for migrating all `img` tags :)